.changeset/gentle-needles-own.md

```diff
@@ -0,0 +1,11 @@
+---
+"@fluid-private/test-end-to-end-tests": minor
+"@fluid-experimental/tree": minor
+---
+---
+"section": legacy
```

Comment:
Why legacy?

Reply:
Updated to `deprecated`, but wasn't sure if this is correct. Would `other` be more appropriate since we are not explicitly adding deprecated tags?